### PR TITLE
[FIX] Radviz: Enable projection for less than two selected variables

### PR DESCRIPTION
**Pull request by VesnaT**
##### Issue

If selecting less than two features (in features list box) to create projection, a warning is shown, which indicates inadequate setting and therefore should not be saved.

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by janezd** [comment]
I'm not sure this is the expected behavior. When the user removes all attributes (or all except one), the widget shows a warning but keeps showing the last valid visualization. The graph should be cleared.

On the other hand, why not showing a radviz with a single attribute (data instances are spread between the center and the anchor) or without any attributes (all data instances are in the center)? Of course if this doesn't cause any other problems down the road.

I lean towards the second alternative but I'm happy with both. But if you opt for the first, I think you should clear the graph, and probably also set `self.selected_vars` to `[]`.

**Comment by VesnaT** [comment]
I chose the second one to be consistent with Linear Projection.

**Comment by janezd** [comment]
What about also removing `no_features = widget.Msg("At least three numeric or categorical variables are required")`? It can work with one, while with 0 it would be just empty.

**Comment by VesnaT** [comment]
No, I added another fix and should be checked.

File: Orange/widgets/visualize/owradviz.py [hunk 1]
```diff
@@ -347,6 +335,12 @@ def colors_changed(self):
 
     def set_data(self, data):
         super().set_data(data)
+        self._init_vizrank()
+        self.init_projection()
+
+    def use_context(self):
```
[the remark this answers is not in the document]
Comment (VesnaT) [reply Orange/widgets/visualize/owradviz.py]: Probably not. It's called from parent's set_data(), after opening the context and before setting self.__invalidated.

[the remark this answers is not in the document]
Comment (janezd) [reply Orange/widgets/visualize/owradviz.py]: You're right.

I searched for `use_context` with PyCharm's "Find in Path", but it didn't find it. Now I see it.

**Pull request merged**